modules/core/include/opencv2/core/quaternion.hpp

```diff
@@ -182,6 +252,39 @@ class Quat
      */
     static Quat<_Tp> createFromRvec(InputArray rvec);
 
+     /**
+     * @brief
+     * from Euler angles
+     *
+     * A quaternion can be generated from Euler angles by combining the quaternion representations of the Euler rotations.
+     *
+     * For example, if we use intrinsic rotations in the order of X-Y-Z,\f$\theta_1 \f$ is rotation around the X-axis, \f$\theta_2 \f$ is rotation around the Y-axis,
+     * \f$\theta_3 \f$ is rotation around the Z-axis. The final quaternion q can be calculated by
+     *
+     * \f[ {q} = q_{X, \theta_1}  q_{Y, \theta_2} q_{Z, \theta_3}\f]
+     * where \f$ q_{X, \theta_1} \f$ is created from @ref createFromXRot,  \f$ q_{Y, \theta_2} \f$ is created from @ref createFromYRot,
+     *  \f$ q_{Z, \theta_3} \f$ is created from @ref createFromZRot.
+     * @param angles the Euler angles in a vector of length 3
+     * @param eulerAnglesType the convertion Euler angles type
+     */
+    static Quat<_Tp> createFromEulerAngles(const Vec<_Tp, 3> &angles, EulerAnglesType eulerAnglesType);
+
+    /**
+     * @brief get a quaternion from a rotation about the Y-axis by \f$\theta\f$ .
+     * \f[q = \cos(\theta/2)+0 i+ sin(\theta/2) j +0k \f]
+     */
+    static Quat<_Tp> createFromYRot(const _Tp theta);
```

Comment:
`angle` or `angle / 2` ? (need to state this clear in docs)

Reply:
Thanks, `angle * 0.5f` should be used here. I will correct this code.